src/units.cpp

```diff
@@ -157,22 +157,17 @@ bool Units::UnitsImpl::isBaseUnit(const std::string &name) const
  *
  * @param multiplier The multiplier to find.
  * @param units The units to find the multiplier for.
- * @param uExp The exponential of the units.
- * @param logMult The log multiplier of the units.
  * @param direction The direction to update multiplier. Either 1 or -1.
  *
  * @return Either @c true or @c false, depending if the units were successfully updated.
  */
-bool updateUnitMultiplier(double &multiplier,
-                          const UnitsPtr &units,
-                          double uExp, double logMult,
-                          int direction)
+
+bool updateUnitMultiplier(double &multiplier, const UnitsPtr &units, int direction)
```

Comment:
Have the output parameter last, this is standard practice for C++

Reply:
... so the `multiplier` at the end?  Is that what you mean?  or the `update` bool last in the comments?  Sorry, I just deleted the middle ones, so I'm not sure which bit to fix!

Reply:
OK, got it, thanks for doing that.

Reply:
np